distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+|No
+
+|Https Protocols
+|https.protocols
+|String
+|Protocols to allow for secure connections
+|TLSv1.1,TLSv1.2
+|No
+
+|Allow Basic Auth Over Http
+|org.codice.allowBasicAuthOverHttp
+|Boolean
+|Set to true to allow Basic Auth credentials to be sent over HTTP unsecurely. This should only be done in a test environment. These events will be audited.
+|false
+|Yes
+
+|Restrict the Security Token Service to allow connections only from DNs matching these patterns
+|ws-security.subject.cert.constraints
+|String
+|Set to a comma separated list of regex patterns to define which hosts are allowed to connect to the STS
+|.*
+|Yes
+
+6+^h|XML Settings
+
+|Parse XML documents into DOM object trees
+|javax.xml.parsers.DocumentBuilderFactory
+|String
+|Enables Xerces-J implementation of `DocumentBuilderFactory`
+|org.apache.xerces.jaxp.DocumentBuilderFactoryImpl
+|Yes
+
+6+^h|File Upload Settings
+
+|File extensions flagged as potentially dangerous to the host system or external clients
+|bad.file.extensions
+|String
+|Files uploaded with these bad file extensions will have their file names sanitized before being saved
+|.exe,
+.jsp,
+.html,
+.js,
+.php,
+.phtml,
+.php3,
+.php4,
+.php5,
+.phps,
+.shtml,
+.jhtml,
+.pl,
+.py,
+.cgi,
+.msi,
+.com,
+.scr,
+.gadget,
+.application,
+.pif,
+.hta,
+.cpl,
+.msc,
+.jar,
+.kar,
+.bat,
+.cmd,
+.vb,
+.vbs,
+.vbe,
+.jse,
+.ws,
+.wsf,
+.wsc,
+.wsh,
+.ps1,
+.ps1xml,
+.ps2,
+.ps2xml,
+.psc1,
+.psc2,
+.msh,
+.msh1,
+.msh2,
+.mshxml,
+.msh1xml,
+.msh2xml,
+.scf,
+.lnk,
+.inf,
+.reg,
+.dll,
+.vxd,
+.cpl,
+.cfg,
+.config,
+.crt,
+.cert,
+.pem,
+.jks,
+.p12,
+.p7b,
+.key,
+.der,
+.csr,
+.jsb,
+.mhtml,
+.mht,
+.xhtml,
+.xht
+|Yes
+
+|File names flagged as potentially dangerous to the host system or external clients
+|bad.files
+|String
+|Files uploaded with these bad file names will have their file names sanitized before being saved
+|crossdomain.xml,
+clientaccesspolicy.xml,
+.htaccess,
+.htpasswd,
+hosts,
+passwd,
+group,
+resolv.conf,
+nfs.conf,
+ftpd.conf,
+ntp.conf,
+web.config,
+robots.txt
+|Yes
+
+|Mime types flagged as potentially dangerous to external clients
+|bad.mime.types
+|String
+|Files uploaded with these mime types will be rejected from the upload
+|text/html,
+text/javascript,
+text/x-javascript,
+application/x-shellscript,
+text/scriptlet,
+application/x-msdownload,
+application/x-msmetafile
+|Yes
+
+|===
+
+These properties are available to be used as variable parameters in input url fields within the ${admin-console}.
+For example, the url for the local csw service (\${secure_url}/services/csw) could be defined as:
+
+[source]
+----
+${variable-prefix}org.codice.ddf.system.protocol}${variable-prefix}org.codice.ddf.system.hostname}:${variable-prefix}org.codice.ddf.system.port}${variable-prefix}org.codice.ddf.system.rootContext}/csw
+----
+
+This variable version is more verbose, but will not need to be changed if the system `host`, `port` or `root` context changes.
+
+
+[IMPORTANT]
```

Comment:
This warning doesn't belong here. This used to be in the section that described `.cfg` files but since it looks like that section is now gone, this warning can be deleted.